File: src/app/models/dto/teacher-submissionDTO.ts
export interface TeacherSubmissionDTO {
  studentId: string;
  studentName: string;
  submissionId?: string;
  fileUrl?: string;
  submissionDate?: string;
  isLate: boolean;
  grade?: number;
  gradeId?: string;
  status: 'PENDIENTE' | 'CORREGIDO' | 'SIN_ENTREGAR';
  activityName: string;
}

File: src/main/java/pepedevelopers/cursitu/model/dto/TeacherSubmissionDTO.java
package pepedevelopers.cursitu.model.dto;

public record TeacherSubmissionDTO(
  String studentId,
  String studentName,
  String submissionId,
  String fileUrl,
  String submissionDate,
  Boolean isLate,
  float grade,
  String status,
  String activityName
) {}
